### ISSUE TITLE
[Feature Request] Enable access to docs after forgetting key, even if that means deleting all docs data.

### ISSUE THREAD

**Issue by thepragmaticmero** [cryptee/web-client#223]
**Is your feature request related to a problem? Please describe.**
I forgor 💀 my encryption key, but I didn't had any data on that account yet anyway. The thing is, I can't "reset" or "delete all data" to use the same mail account, there's not an option inside crypt.ee/account for it, I like my cute paid protonmail account and I would like to use it. Is there a workaround? I could delete my account and create another one with the same email, I just don't know if my email would be blocked by the crypt.ee account system after destroying it. Any recommendations?

**Describe the solution you'd like**
Add a "I forgot my encryption key" option on the "login & security > change encryption key > " menu, and warn the user that all the docs data will be destroyed.

**Additional context**
Largest human mattress dominoes :D

**Comment by johnozbay** [comment]
Hi there! 
Happy to have you on board Cryptee! 

First. let's get the most important and critical thing out of the way. 

> Additional context
> Largest human mattress dominoes :D

https://www.youtube.com/watch?v=TE5RdFFgW0w

---

Now, onto the significantly less serious topic of deleting accounts! 😅

> I could delete my account and create another one with the same email, I just don't know if my email would be blocked by the crypt.ee account system after destroying it. Any recommendations?

TLDR; You can actually delete your account without your encryption key, and immediately re-use the same email to sign up if you wish. Our system won't block you.

---

Longer answer : 

You can actually delete your account without your encryption key, and this is one of the main reasons why on Cryptee your encryption key and password are separate and independent from one another. So that you can login with your email+pass and delete your account even if you forget your key. 

To do this you can head over to 'settings', and you'll see a delete account button at the bottom of the list. 

Once you delete your account all your data and account details and everything is deleted immediately from our servers, so you can use the same email address right away if you wish. 

Our system will only block the email address if it detects storage abuse. i.e. if you store tons of data, delete, tons of data, delete, create, delete, create delete too many times in a row (e.g. hundreds of times in a super short period of time) then our system may flag your account depending on a few other parameters we check to prevent abuse / attacks etc. 

So you should be all set and shouldn't have any issues. If you experience any issues I'd recommend pinging our helpdesk, as they're better equipped to help with account specific issues you may experience. 

---

And since users can already delete their accounts and start over with one click, we figured it's just as easy to do that vs create a separate (yet duplicate feature) that allows resetting the account in the even of someone forgetting their encryption key. 

Hoping this makes sense and helps!

Best,

J

**Comment by thepragandsensdiary** [comment]
Hi there, the E-stonian man himself haha, thanks for the answer, sorry for the duplicate, I tried to find a similar problem on the issues/discussions but I searched using the wrong keywords. Anyways, I'm restructuring all my accounts for a better thread model.
Its a relief that my account isn't nuked haha, thanks for the guide~ Great fan of your work~ 
Kindly: A fan of playing piano, black color minimalism and software engineering~

**Comment by johnozbay** [comment]
Hey there @thepragandsensdiary! 

Happy to hear I could be of help, and thanks a million for the kind words. 🙏🏻 
Really happy to hear you like my music as well. I love when my music/software worlds collide 😅

It's a smart move to re-structure accounts for a new threat model. I wish more people did this. 
We all could use a good and thorough spring-cleaning so to speak!

All the very best from a snowy winter wonderland Tallinn, Estonia,

![image](https://github.com/user-attachments/assets/d9bdbd3d-08f5-4cdc-ac63-385866a2dc54)
